### ISSUE TITLE
DHCP problem

### ISSUE THREAD

**Issue by Onepamopa** [AdguardTeam/AdGuardHome#1179]
DHCP server does not see ethX:Y interfaces.

I have eth1 10.0.0.1 (PPPoE server) and eth1:1 10.0.1.1  (both static IPs), AdGuard does not see eth1:1, if I set DHCP server to eth1 - it sets:
> cat /etc/dhcpcd.conf
interface eth1
static ip_address=10.0.0.1/24
static domain_name_servers=10.0.0.1

The web interface is configured to 10.0.1.1 it saves 10.0.0.1  .... WTF ?

**Comment by Onepamopa** [comment]
AdGuardHome.yaml:
dhcp:
  enabled: true
  interface_name: eth1
  gateway_ip: 10.0.1.1
  subnet_mask: 255.255.255.0
  range_start: 10.0.1.2
  range_end: 10.0.1.254
  lease_duration: 86400
  icmp_timeout_msec: 1000

**Comment by Onepamopa** [comment]
Additionally - /etc/dhcpcd.conf does not exist in my system, AdGuard did not automatically create it. I had to touch /etc/dhcpcd.conf or AdGuard wouldn't start at all.

**Comment by Onepamopa** [comment]
Additionally - EDIT button on the static DHCP leases would be nice.

**Comment by szolin** [comment]
How did you enable DHCP server?  Did you use web interface or did you modify yaml file directly?

> /etc/dhcpcd.conf does not exist in my system
Most probably because your OS doesn't have the necessary packages installed.

> I had to touch /etc/dhcpcd.conf
This is wrong, please revert your system configuration back as it was!

**Comment by Onepamopa** [comment]
@szolin I enabled via web interface, DHCP is working (adding correct IPs from 10.0.1.0/24) but /etc/dhcpcd.conf shows 10.0.0.0/24 
What packages do you suggest I install ?

**Comment by szolin** [comment]
> /etc/dhcpcd.conf shows 10.0.0.0/24

Did you revert the changes you made to the system (`had to touch /etc/dhcpcd.conf`) prior enabling DHCP server?

**Comment by Onepamopa** [comment]
I'll do that tonight and let you know.

**Comment by Onepamopa** [comment]
I've deleted /etc/dhcpcd.conf, AdGuard started without it (no need for creation of that file).
Right now:
2019/11/23 13:38:27 [info] DHCP: loaded 21 leases from DB
2019/11/23 13:38:27 [info] DHCP: listening on 0.0.0.0:67

Even though I've set eth1 (which has 2 addresses: 10.0.0.1[pppoe] and 10.0.1.1[dhcp]).
The DHCP server is configured like:
eth1 - 10.0.1.1
GW 10.0.1.1
Start 10.0.1.2
End 10.0.1.254

Why would AdGuard DHCP listen on 0.0.0.0 ?? It should listen on the "GW" address.
